storage-proofs/src/example_helper.rs

```diff
@@ -337,10 +337,14 @@ pub trait Example<'a, C: Circuit<Bls12>>: Default {
         let (data_size, challenge_count, m, sloth_iter, typ) = {
             let matches = instance.clap();
 
-            let data_size = value_t!(matches, "size", usize).unwrap() * 1024;
-            let challenge_count = value_t!(matches, "challenges", usize).unwrap();
-            let m = value_t!(matches, "m", usize).unwrap();
-            let sloth_iter = value_t!(matches, "sloth", usize).unwrap();
+            let data_size = value_t!(matches, "size", usize)
+                .map(|size| size * 1024)
+                .expect("Failed to parse `size` CLI arg as `usize`");
```

Comment:
Question out of Rust inexperience: is it the same to have the check after the `map`?

Reply:
It is, as `map` only gets executed when no error occurs